Jellyfin.Plugin.AniDB/Configuration/PluginConfiguration.cs

```diff
@@ -1,4 +1,4 @@
-﻿using MediaBrowser.Model.Plugins;
+using MediaBrowser.Model.Plugins;
```

Comment:
does this add or remove a BOM?

Reply:
Removes, should be reverted.

Reply:
I didn't go that far with cleaning. I think the BOM should be kept in there for now, and cleaned and removed for all files later in a separate PR, so it's universal for all files.

Reply:
fine by me :+1: 

*and sorry didn't intent to post this as a single comment :sweat_smile: